### PR TITLE
Fix healthcheck stage

### PR DESCRIPTION
**Pull request by gquadrati**
#### List of Changes

Fix healthcheck stage

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by fabriziopapi — APPROVED** [review]

File: .devops/deploy-pipelines.yml [hunk 1]
```diff
@@ -130,19 +133,17 @@ stages:
               slotName: 'production'
             displayName: Deploy to io-p-fn-elt slot
 
-  # Check that the io-p-fn-elt instance is healthy
+  # Check that the staging instance is healthy
   - stage: Healthcheck
+    pool:
+      name: io-prod-linux
```
Comment (fabriziopapi) [inline .devops/deploy-pipelines.yml:139]:
```suggestion
      name: $(AGENT_POOL)
```
this could be replaced with env var

**Review by fabriziopapi — CHANGES_REQUESTED** [review]

File: .devops/deploy-pipelines.yml (hunk 1 above)
Comment (gquadrati) [reply .devops/deploy-pipelines.yml:139]: The first time we run it's necessary to set it as string. Unfortunately, I wasn't able to make it work

Comment (fabriziopapi) [reply .devops/deploy-pipelines.yml:139]: You are right, but we can do it without merging the literal value:
https://pagopa.atlassian.net/wiki/spaces/IC/pages/84902884/Troubleshooting#L%E2%80%99healthcheck-non-funziona%2C-errore%3A-Pipeline-does-not-have-permissions-to-use-the-referenced-pool(s)-io-prod-linux.-For-authorization-details%2C-%E2%80%A6

Comment (gquadrati) [reply .devops/deploy-pipelines.yml:139]: I know, I wrote it :)

Comment (gquadrati) [reply .devops/deploy-pipelines.yml:139]: The problem is I wasn't able to run it successfully on this branch.

Comment (fabriziopapi) [reply .devops/deploy-pipelines.yml:139]: ok, let's do it with 2 PR then!

**Comment by fabriziopapi** [comment]
This should be merged after https://github.com/pagopa/io-functions-elt/pull/47

**Review by fabriziopapi — APPROVED** [review]

**Pull request merged**